CSP_Minizinc_Code.py

```diff
@@ -3,44 +3,48 @@
 from abc import ABC,abstractmethod
 from custom_dtypes import *
 from minizinc import Instance, Model, Solver
+import argparse
 
 #Provide the contextual information needed for selection and deployment.
 class Context_Monitor:
     def __init__(self,repo_image):
         #Initially the robot's context monitor is null
         self.__gripper_status = False
         self.__robot_in_motion = False
-        self.__finger_open = False
         self.__time_stamp = 0.0
         self.__repo_image=repo_image
 
     def get_robot_status(self):
         #Robot senses the current state of its environment
         #Initially the sensed data is provided explicitly
         self.__gripper_status=True
-        self.__robot_in_motion=False
-        self.__finger_open=True
+        self.__robot_in_motion=True
 
     def update_info_to_repo(self):
         #Updates the current state of the robot to the repository
-        self.__repo_image.update_context(self.__gripper_status,self.__robot_in_motion,self.__finger_open)
+        self.__repo_image.update_context(self.__gripper_status,self.__robot_in_motion)
 
 
 #Repository contains information for deployment. It is the central component in the architecture. Every components updates the information to the repository. 
 class Repository():
     def __init__(self, active_safety_monitor = SafetyMonitor.NO_SELECTION, active_sensor = Sensors.NO_SELECTION,
-                 current_context = [False, False, False],fixed_deployment = True,
+                 current_context = [False, False],fixed_deployment = True,
                  safety_monitor_platform = Platforms.NO_SELECTION):
 
         self.__active_safety_monitor = active_safety_monitor
         self.__active_sensor = active_sensor
         self.__safety_monitor_platform = safety_monitor_platform
         self.__current_context = current_context
         self.__fixed_deployment = fixed_deployment
+        self.min_force_sensor_count = 1

```

Comment:
This is actually presence of force sensor but we changed it to min_force_sensor_count to make our MiniZn model working. We'll have to look for possibilities to modify this